meta-lmp-base/recipes-sota/lmp-device-register/lmp-device-register_git.bb

```diff
@@ -4,18 +4,16 @@ LIC_FILES_CHKSUM = "file://COPYING.MIT;md5=838c366f69b72c5df05c96dff79b35f2"
 
 DEPENDS = "boost curl glib-2.0"
 
-SRCREV = "633b11b397b996f2bce2438324db203879b9ffda"
+SRCREV = "1660bfb9450b9ffc6178b777f8e074f3af9204af"
 
 SRC_URI = "git://github.com/foundriesio/lmp-device-register.git;protocol=https"
 
 # Default to master tag
 LMP_DEVICE_REGISTER_TAG ?= "master"
 
-APP_TYPE ?= "${@'DOCKER_COMPOSE_APP' if d.getVar('DOCKER_COMPOSE_APP') == '1' else 'DOCKER_APPS'}"
-
 PACKAGECONFIG ?= "aklitetags dockerapp"
 PACKAGECONFIG[aklitetags] = "-DAKLITE_TAGS=ON -DDEFAULT_TAG=${LMP_DEVICE_REGISTER_TAG},-DAKLITE_TAGS=OFF,"
-PACKAGECONFIG[dockerapp] = "-D${APP_TYPE}=ON,-D${APP_TYPE}=OFF,"
+PACKAGECONFIG[dockerapp] = "-DDOCKER_COMPOSE_APP=ON,-DDOCKER_COMPOSE_APP=OFF,"
```

Comment:
As we're removing references for docker app, I think it would also make sense to rename this packageconfig variable to composeapp, just to avoid confusion.

Reply:
done